integration-tests/jsonata/src/main/resources/spec/expressions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": FirstName & " " & Surname,
+    "mobile": Phone[type = "mobile"].number
```

Comment:
I see that this is supposed to be some sort of JSONata transformation recipe, but I still wonder whether it is valid JSON?

Reply:
It's not valid JSON. The best term I could find is "spec" from the camel-jsontata documentation, so I would propose to rename it to expressions.spec then. What do you think ?

Reply:
Yes, I think using somethig else than .json would be less misleading for both users and IDEs/editors :)